app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -186,15 +189,25 @@ def track_events
       track_billing_events
     end
 
-    def redirect_user(redirect_uri)
-      case IdentityConfig.store.openid_connect_redirect
-      when :client_side
+    # rubocop:disable Layout/LineLength
+    def redirect_user(redirect_uri, user_uuid)
+      server_side_uuid = IdentityConfig.store.openid_connect_redirect_server_side_uuids.include?(user_uuid)
+      client_side_uuid = IdentityConfig.store.openid_connect_redirect_client_side_uuids.include?(user_uuid)
+      client_side_js_uuid = IdentityConfig.store.openid_connect_redirect_client_side_js_uuids.include?(user_uuid)
+
+      default = IdentityConfig.store.openid_connect_redirect
```

Comment:
it's potentially more error-prone in some ways, but if we had one config that was a hash/dictionary, we could just do one lookup and simplify a lot of this code?

```
config = {
  "uuid-a" => "server_side",
  "uuid-b" => "client_side",
  # etc....
}.transform_values(&:to_sym)

case config.fetch(current_user.uuid, IdentityConfig.store.openid_connect_redirect)
```

Reply:
I like that better

Reply:
I opted to switch the values to strings though